src/WebSocketClient.ts

```diff
@@ -8,7 +8,7 @@ import type { ContextTimed, ContextTimedInput } from '@matrixai/contexts';
 import { AbstractEvent } from '@matrixai/events';
 import { createDestroy } from '@matrixai/async-init';
 import Logger from '@matrixai/logger';
-import WebSocket from 'ws';
+import * as ws from 'ws';
```

Comment:
When we use in the browser, don't these imports not work? Or is it expected that one uses a bundler.

Reply:
I've only tested with a bundler. But the only conditions we'll use js-ws under is one with a bundler.